modules/custom/inope/inope.install

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * @file
+ * Install file for iNope.
+ */
+
+/**
+ * Enables the iNope Basic Configuration and Features.
+ */
+function inope_install() {
+  module_enable(array(
```

Comment:
OK - i think so -- just make sure https://www.drupal.org/project/master is in your make file and then add the modules you'd like enabled to the settings.php file -- you should see the place, if not ping me.